optax/losses/_self_supervised_test.py

```diff
@@ -124,6 +124,362 @@ def test_vmap(self, anchor, positive, negative):
     np.testing.assert_allclose(vmap_loss.flatten(), original_loss.flatten()
                                , atol=1e-4)
 
+class ByolLossTest(absltest.TestCase):
+
+  def test_symmetric_batched_matches_handmade(self):
```

Comment:
rewrite as a random inputs test parameterized by a random seed please (instead of hard-coding the numbers)